src/components/menu/components/dishes/components/categoryDishes/components/dish/Dish.js

```diff
@@ -0,0 +1,66 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import dishReader from './readers/dishReader';
+import _noop from 'lodash/noop';
+
+//Icons
+import rupee from '../../../../../../../../images/rupee.svg';
+
+//Components
+import DishImage from '../../../../../../../../commonComponents/dishImage';
+import DishButtton from './components/dishButtton';
+
+//CSS
+import './dish.css';
+
+function Dish(props) {
+  const { dish, quantities, handleEvents } = props;
+  const dishId = dishReader.id(dish);
+  const name = dishReader.name(dish);
+  const price = dishReader.price(dish);
+  const isVeg = dishReader.isVeg(dish);
+  const description = dishReader.description(dish);
+
+  return (
+    <div className='dish'>
+      <div>
+        <DishImage isVeg={isVeg} />
+        <h3 className='styledH3'>{name}</h3>
+        <p className='styledP'>
+          <img src={rupee} alt='rupee' className='icon' /> {price}
+        </p>
+        <div className='styledDiv'>{description}</div>
+      </div>
+      <div>
+        <DishButtton
```

Comment:
i think you are rendering dish button here only and for plus minus button you are directly rendering it in cartItem component directly so you can directly render Button here